tools/build/npm/templates/deploy.py

```diff
@@ -43,7 +43,7 @@
 
 npm_registry = npm_repositories[repo_type]
 
-npm_token = os.getenv('DEPLOY_NPM_EMAIL')
+npm_token = os.getenv('DEPLOY_NPM_TOKEN')
```

Comment:
@ljupcovangelski no idea how we missed this, but this should explain why releases of the npm libraries don't work